packages/core/redux/voting/actions.js

```diff
@@ -83,20 +90,23 @@ export const createPoll = (poll: PollDetailsModel) => async (dispatch) => {
 
   try {
     dispatch(handlePollCreated(stub))
+    dispatch(goToVoting())
     const dao = await contractsManagerDAO.getVotingManagerDAO()
     const transactionHash = await dao.createPoll(poll.poll())
     dispatch(handlePollRemoved(stub.id()))
     dispatch(handlePollUpdated(stub.transactionHash(transactionHash)))
   } catch (e) {
-    // eslint-disable-next-line
-    console.error('create poll error', e.message)
     dispatch(handlePollRemoved(stub.id()))
   }
 }
 
-export const removePoll = (poll: PollDetailsModel) => async (dispatch) => {
+export const removePoll = (pollObject: PTPoll) => async (dispatch, getState) => {
+  let poll = pollObject && pollObject.id
+    ? getState().get(DUCK_VOTING).list().item(pollObject.id)
```

Comment:
`getState().get(DUCK_VOTING)` must be replaced with appropriate selector from `selectors/models.js`.